ci_framework/roles/edpm_deploy_baremetal/tasks/main.yml

```diff
@@ -182,6 +182,20 @@
           --type json
           -p '[{"op": "replace", "path": "/spec/services/0", "value": "repo-setup-downstream"}]'
 
+- name: Patch OpenStackDataPlaneNodeSet resource to include content provider insecure registry
```

Comment:
ok, this is needed because edpm_deploy_baremetal was not updated to use kustomize plugin.